server/commands/userCreator.ts

```diff
@@ -48,6 +48,8 @@ export default async function userCreator({
       {
         model: User,
         as: "user",
+        where: { teamId },
+        required: true,
```

Comment:
🥦 3/ only consider users from the target `teamId`. This opens the door to having the same external `providerId` on multiple users on multipole teams